src/Discord/Parts/Channel/Message.php

```diff
@@ -670,4 +670,12 @@ public function getRepositoryAttributes(): array
             'channel_id' => $this->channel_id,
         ];
     }
+    
+    /**
+     * {@inheritdoc}
+     */
+    public function link(): string
```

Comment:
```suggestion
    public function getLinkAttribute(): string
```
I think this would work better as an attribute, what do you think?

Reply:
Done